frontend/pages/model/[modelId]/access-request/schema.tsx

```diff
@@ -1,116 +1,17 @@
-import { Schema } from '@mui/icons-material'
-import ArrowBack from '@mui/icons-material/ArrowBack'
-import { Box, Button, Card, Container, Grid, Stack, Typography } from '@mui/material'
-import { useGetSchemas } from 'actions/schema'
 import _ from 'lodash-es'
 import { useRouter } from 'next/router'
-import { useCallback, useMemo, useState } from 'react'
-import EmptyBlob from 'src/common/EmptyBlob'
-import Loading from 'src/common/Loading'
 import Title from 'src/common/Title'
-import MultipleErrorWrapper from 'src/errors/MultipleErrorWrapper'
-import Link from 'src/Link'
-import SchemaButton from 'src/schemas/SchemaButton'
-import { SchemaInterface, SchemaKind } from 'types/types'
+import SchemaSelect from 'src/schemas/SchemaSelect'
+import { SchemaKind } from 'types/types'
 
 export default function AccessRequestSchema() {
   const router = useRouter()
   const { modelId }: { modelId?: string } = router.query
```

Comment:
You could use this ID to fetch the model like so:
```
const { model, isModelLoading, isModelError } = useGetModel(modelId, EntryKind.MODEL)
```
You'll then need to add back in the error handling:
```
  const error = MultipleErrorWrapper(`Unable to load schema page`, {
    isModelError,
  })
  if (error) return error
```
And finally you'll need to manage the loading state just below the `Title`:
```
      <Title text='Select a schema' />
      {isModelLoading && <Loading />}
```
You can then pass the `model` into `SchemaSelect` via the `entry` prop.
```
<SchemaSelect schemaKind={SchemaKind.ACCESS_REQUEST} entry={model} />
```